cypress/e2e/gui/customers.cy.js

```diff
@@ -0,0 +1,299 @@
+          {
+            fixture: 'customers/logisticsCustomers',
+          }
+        ).as('getLogisticsCustomers');
+        cy.findByTestId('industry-filter').select('Logistics');
+        cy.wait('@getLogisticsCustomers');
+
+        cy.get('tbody tr').should('have.length', 1);
+      });
+
+      it('filters the customers by retail industry', () => {
+        cy.intercept('GET', `${CUSTOMERS_URL}?page=1&limit=10&size=All&industry=Retail`, {
+          fixture: 'customers/retailCustomers',
+        }).as('getRetailCustomers');
+        cy.findByTestId('industry-filter').select('Retail');
+        cy.wait('@getRetailCustomers');
+
+        cy.get('tbody tr').should('have.length', 1);
+      });
+
+      it('filters the customers by technology industry', () => {
+        cy.intercept(
+          'GET',
+          `${CUSTOMERS_URL}?page=1&limit=10&size=All&industry=Technology`,
+          {
+            fixture: 'customers/technologyCustomers',
+          }
+        ).as('getTechnologyCustomers');
+        cy.findByTestId('industry-filter').select('Technology');
+        cy.wait('@getTechnologyCustomers');
+
+        cy.get('tbody tr').should('have.length', 3);
+      });
+
+      it('filters the customers by HR industry', () => {
+        cy.intercept('GET', `${CUSTOMERS_URL}?page=1&limit=10&size=All&industry=HR`, {
+          fixture: 'customers/HRCustomers',
+        }).as('getHRCustomers');
+        cy.findByTestId('industry-filter').select('HR');
+        cy.wait('@getHRCustomers');
+
+        cy.get('tbody tr').should('have.length', 4);
+      });
+
+      it('filters the customers by finance industry', () => {
+        cy.intercept(
+          'GET',
+          `${CUSTOMERS_URL}?page=1&limit=10&size=All&industry=Finance`,
+          {
+            fixture: 'customers/financeCustomers',
+          }
+        ).as('getFinanceCustomers');
+        cy.findByTestId('industry-filter').select('Finance');
+        cy.wait('@getFinanceCustomers');
+
+        cy.get('tbody tr').should('have.length', 6);
+      });
+
+      it('filters the customers by all industries', () => {
+        cy.findByTestId('industry-filter').select('Technology');
+        cy.intercept('GET', `${CUSTOMERS_URL}?page=1&limit=10&size=All&industry=All`, {
+          fixture: 'customers/allCustomers',
+        }).as('getAllCustomers');
+        cy.findByTestId('industry-filter').select('All');
+        cy.wait('@getAllCustomers');
+
+        cy.get('tbody tr').should('have.length', 9);
+      });
+    });
+  });
+});
+
+describe('Accessibility', () => {
+  beforeEach(() => {
+    cy.setCookie('cookieConsent', 'accepted');
+    cy.visit('/');
+    cy.injectAxe();
+  });
+
+  context('Light Mode', () => {
+    it('finds no a11y issues in light mode in the customer table', () => {
+      cy.checkA11y();
+    });
+
+    it('finds no a11y issues in the customer details and address view', () => {
+      cy.get('button').contains('View').click();
+      cy.checkA11y();
```

Comment:
Que tal verificar que o tema mudou antes da verificação de acessibilidade?

Algo assim: `cy.get('[data-theme="dark"]').should('exist')`.